AppLovin MAX Demo App - ObjC/AppLovin MAX Demo App - ObjC/Base Classes/ALHomeViewController.m

```diff
@@ -87,16 +87,18 @@ - (void)openSupportSite
     {
         SFSafariViewController *safariController = [[SFSafariViewController alloc] initWithURL: [NSURL URLWithString: kSupportLink]
                                                                        entersReaderIfAvailable: YES];
-        [self presentViewController: safariController animated: YES completion:^{
-            [[UIApplication sharedApplication] setStatusBarStyle: UIStatusBarStyleDefault];
-        }];
+        [self presentViewController: safariController animated: YES completion:^{}];
     }
     else
     {
         [[UIApplication sharedApplication] openURL: [NSURL URLWithString: kSupportLink]];
     }
 }
 
+- (UIStatusBarStyle) preferredStatusBarStyle {
```

Comment:
1. Formatting (@Vedant-Mehta39 please brief Andy on the importance of consistency codebase formatting).
2. Since this is an override of a base `UIViewController` method, convention is to declare it at the top of file, before `- (void)viewDidLoad`.
3. How does it look with and without dark mode? Could you provide screenshots?